addons/payment/views/payment_acquirer_views.xml

```diff
@@ -89,10 +89,7 @@
                                            placeholder="Select countries. Leave empty to make available everywhere."
                                            options="{'no_open': True, 'no_create': True}"/>
                                 </group>
-                                <group string="Payment Followup" name="payment_followup">
-                                    <field name="journal_id" context="{'default_type': 'bank'}"
-                                           attrs="{'required': [('state', '!=', 'disabled'), ('provider', 'not in', ['none', 'transfer'])]}"/>
-                                </group>
+                                <group string="Payment Followup" name="payment_followup" invisible="1"/>
```

Comment:
Shouldn't this be moved altogether to `account_payment`?

Reply:
```xml
            <group name="payment_followup" position="attributes">
                <attribute name="attrs">
                    {'invisible': [('provider', '=', 'transfer')]}
                </attribute>
            </group>
```
in payment_transfer :D